### PR TITLE
feat: Resource details page

### PR DESCRIPTION
**Pull request by danielptv**
[ODDSK-189](https://h-ka-team-rdqzrlfpomci.atlassian.net/browse/ODDSK-189).

Examples: 
* [Bar chart](https://deploy-preview-99--open-data-dashboard.netlify.app/en/view/Wohnberechtigte-Bev%C3%B6lkerung-1?search=%7B%22Jahr%22%3A%7B%22min%22%3A%222009%22%2C%22max%22%3A%222009%22%7D%7D&visualization=barChart)
* [Map](https://deploy-preview-99--open-data-dashboard.netlify.app/en/view/points-of-interest-kinos-3)
* [Embedded](https://deploy-preview-99--open-data-dashboard.netlify.app/en/view/karlsruher-mietspiegel-2023-2)

Currently, the details views are available under `/view` route. I'm personally not happy with the naming but couldn't come up with anything better on the fly.

### CODE REVIEW

File: src/messages/de.json [hunk 1]
```diff
@@ -52,6 +52,16 @@
...
         "resetTooltip": "Ansicht zurücksetzen"
     },
+    "ResourceDetailsControls": {
+        "download": "Herunterladen",
+        "embed": "Einbetten",
+        "embedBody": "Die Visualisierung dieser Ressource kann mithilfe des folgenden Codes eingebettet werden.",
+        "embedHeight": "Höhe",
+        "embedTitle": "Ressourcendarstellung einbetten",
+        "embedWidth": "Breite",
+        "fullscreen": "Vollbildschirm",
```
Comment (Zerdados) [inline src/messages/de.json:62]: Maybe 'Vollbild' instead of 'Vollbildschirm'

**Review by Zerdados — COMMENTED** [review]
Looks good overall.
The only thing I noticed is that we "forget" the state when switching to the fullscreen view ( this of course makes sense since we are simply linking to another page), which becomes noticeable when selecting different yAxes, or switching to the table, in the ChartTable.

**Comment by danielptv** [comment]
> Looks good overall. The only thing I noticed is that we "forget" the state when switching to the fullscreen view ( this of course makes sense since we are simply linking to another page), which becomes noticeable when selecting different yAxes, or switching to the table, in the ChartTable.

The problem here is that you have to use the "Return" button in your browser, meaning you get the page with the query parameters when you left it for fullscreen rather than the updated parameters from the fullscreen page. I dont't think we can do much here. If we would be navigating using a button it wouldn't be a problem to pass the actual query parameters (as you can see when you navigate from details page to fullscreen).

Let me know if you have a suggestion how we could improve the UX here

**Comment by Zerdados** [comment]
> > Looks good overall. The only thing I noticed is that we "forget" the state when switching to the fullscreen view ( this of course makes sense since we are simply linking to another page), which becomes noticeable when selecting different yAxes, or switching to the table, in the ChartTable.
> 
> The problem here is that you have to use the "Return" button in your browser, meaning you get the page with the query parameters when you left it for fullscreen rather than the updated parameters from the fullscreen page. I dont't think we can do much here. If we would be navigating using a button it wouldn't be a problem to pass the actual query parameters (as you can see when you navigate from details page to fullscreen).
> 
> Let me know if you have a suggestion how we could improve the UX here

Unfortunately, I also don't know of a better way to do this right now. Maybe we will find a nice way later down in development, when the Open Data Dashboard is a bit more fleshed out. I think it is okay to leave it as is for now, we should just keep it in mind.

**Review by Zerdados — APPROVED** [review]

**Pull request merged**